package.json

```
@@ -7,12 +7,14 @@
   "scripts": {
     "build": "turbo run build",
     "dev": "turbo run dev --parallel",
+    "translate": "ts-node scripts/translate.ts",
```

```suggestion
    "translate": "ts-node -T scripts/translate.ts",
```

This will run it transpile-only, which is faster